react/AutocompleteBlock.tsx

```diff
@@ -76,7 +77,7 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
       action = success
         ? {
             label: translateMessage(messages.seeCart),
-            href: '/checkout/#/cart',
+            href: checkoutUrl ?? '/checkout/#/cart',
```

Comment:
Might be a good idea to validate for an empty string here

Reply:
Done!